ls1intum/Apollon
# `Development`: Fix alignment of elements to grid issue 2

### Checklist

### Motivation and Context
This PR fixes the issue of elements not aligning perfectly on the grid.
Issue Number: https://github.com/ls1intum/Apollon/issues/279

**Changes include:**
- Adjust the coefficients to produce a multiple of ten (each square on the grid has 10px width & height)
- Fix alignment issue with the following elements:
1. Object in object diagram and communication diagram
2. Actor in a use case
3. Transition and place in a Petri net
4. Input/Output in a flowchart
5. All elements of a class diagram (not mentioned in the ticket but these were also not perfectly aligned)

### Steps for Testing
1. Clone the repo
2. Checkout to branch `enhancement/uml-elements-scaling-fix`
3. Execute commands: `yarn install`, followed by `yarn start`
4. Observe elements mentioned above are aligned with the grid.

### Screenshots
#### Before
![class-develop](https://user-images.githubusercontent.com/36473858/209253984-abd3ab99-205a-4fa7-9baf-7faf37de937c.png)
![others-dev](https://user-images.githubusercontent.com/36473858/209254006-d711812c-e0ce-4fc9-a62f-1e10fa6d009b.png)

#### After
![class-fixed](https://user-images.githubusercontent.com/36473858/209254016-6c903910-6bdb-4404-a3f5-4fca720c9a69.png)
![others-fixed](https://user-images.githubusercontent.com/36473858/209254023-4683a163-09f6-4f46-bcd4-33899b5ee8d3.png)
